web-app/pages/404.tsx

```diff
@@ -1,4 +1,4 @@
-export default function Example() {
+export default function handle404() {
```

Comment:
Even though it looks like we're defining a function here, and functions usually have verbs in their names (like you did with `handle404`), this is a special type of function: a React component.

So it should just be a noun.

And I think generally we'll want to use PascalCase for component names. 

https://airbnb.io/javascript/react/#naming

So maybe `ErrorPage404` would be good to try.